src/app.md

```diff
@@ -325,8 +405,15 @@ fn main() {
 [`File`]: https://doc.rust-lang.org/std/fs/struct.File.html
 [`flate2::read::GzDecoder::new`]: https://docs.rs/flate2/*/flate2/read/struct.GzDecoder.html#method.new
 [`flate2::write::GzEncoder`]: https://docs.rs/flate2/*/flate2/write/struct.GzEncoder.html
+[`std::path::Path::strip_prefix`]: https://doc.rust-lang.org/std/path/struct.Path.html#method.strip_prefix
+[`tar::Archive`]: https://docs.rs/tar/*/tar/struct.Archive.html
+[`tar::Archive::entries`]: https://docs.rs/tar/*/tar/struct.Archive.html#method.entries
 [`tar::Archive::unpack`]: https://docs.rs/tar/*/tar/struct.Archive.html#method.unpack
 [`tar::Builder`]: https://docs.rs/tar/*/tar/struct.Builder.html
+[`tar::Entry`]: https://docs.rs/tar/0.4.13/tar/struct.Entry.html
```

Comment:
please use wildcard link to always point to the latest documentation. Please see links below